Mbarak-Mbigo/cp2-frontend
# Ft login logic

#### What does this PR do?
Add login Logic
#### Description of Task to be completed?
Add login Logic
#### How should this be manually tested?
Start application and logic 
Expected results: Login view should capture errors and login successfully when the right credentials are given
#### Any background context you want to provide?
N/A
